src/Bookings.js

```diff
@@ -1,21 +1,47 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+import FakeBookings from "./data/fakeBookings.json";
+
+
 
 const Bookings = () => {
   const search = searchVal => {
     console.info("TO DO!", searchVal);
   };
+  const [bookings, setBookings] = useState([])
+
+  function doingFetch() {
+    fetch("https://cyf-react.glitch.me")
+      .then((response) => {
+        if (!response.ok) {
+          throw new Error(response.status)
+        } else {
+          return response.json()
+        }
+
+      })
+      .then((data) => {
+        console.log("data----->", data)
```

Comment:
nit: It's usually better to remove the debugging statements before you send your code for review.